docs/migration-guides/v8.0.0.md

```diff
@@ -0,0 +1,128 @@
+# v8.0.0 Migration guide
+
+This guide assumes you have an existing app running on `sku v7. If you are still on a version of sku prior to v7, please checkout the [v7 Migration guide](./migration-guides/v7.0.0) as well.
+
+> If you have any issues or concerns updating to sku v8, please post in the #sku-support slack channel.
+
+**Obligatory warning to check everything still works**
+
+We try to validate sku works across a big variety of scenarios before releasing. However, as with all major sku releases, please validate that your app continues to work as expected in a real environment as well as locally.
+
+## Peer dependencies
+
+`React` is now a `peerDependency` of sku. Ensure your version of React falls within the required range.
+
+> At the time of release, sku requires `React v16.3.0` or above.
+
+## Dynamic imports
+
+If you are utilizing dynamic imports in your app, you may want to consider migrating them to be [loadable-components](https://www.smooth-code.com/open-source/loadable-components/). Dynamic imports should continue to work as before but you may receive benefits by utilizing `loadable`.
+
+See [Code splitting](./docs/code-splitting) for more info on how to do this.
+
+## SkuProvider
+
+The `SkuProvider` is a React component that needs to be rendered at the very top level of the app. This only needs to be done as part of the [render](./docs/building-the-app#render) or [server](./docs/building-the-app#server) entry (depending on the app type), it is **not** required for the [client](./docs/building-the-app#client) entry.
+
+There are further updates to the SSR render API. [See here](#rendercallback).
+
+### Render entry example
+
+```diff
+import React from 'react';
+import { renderToString } from 'react-dom/server';
+import App from './App';
+
+export default {
+-  renderApp: ({ environment, site, route }) =>
++  renderApp: ({ SkuProvider, environment, site, route }) =>
+    renderToString(
++      <SkuProvider>
+        <App environment={environment} site={site} route={route} />
++      </SkuProvider>
+    ),
+
+  renderDocument: ({ app, bodyTags, headTags }) => `
+    <!DOCTYPE html>
+    <html>
+      <head>
+        <meta charset="UTF-8">
+        <title>My Awesome Project</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        ${headTags}
+      </head>
+      <body>
+        <div id="app">${app}</div>
+        ${bodyTags}
+      </body>
+    </html>
+  `
+};
+```
+
+## Client entry
+
+The client entry should now export a function rather than hydrating the app immediately.
+
+```diff
+import React from 'react';
+import { hydrate } from 'react-dom';
+import App from './App/App';
+
++export default () => {
+  hydrate(<App />, document.getElementById('app'));
++};
+```
+
+> Statically rendered apps should checkout the new [provideClientContext](./docs/static-rendering#provideclientcontext) feature, for passing config and other state from render to the client.
+
+## **Static render apps**
```

Comment:
Is this supposed to be formatted differently to that other headings?

Reply:
Yeah it gives some extra formatting in the contents